TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/ImageEditor.js

```diff
@@ -298,9 +298,18 @@ function (Ember, $, FileUpload, template, cropTemplate, BooleanEditor, Spinner,
 		_mediaBrowserView: null,
 
 		_beforeMediaBrowserIsShown: function () {
-			var that = this;
+			var that = this,
+				value = null;
+
+			try {
+				value = JSON.parse(this.get('value'));
+			} catch (exception) {}
```

Comment:
It would only fail if the value is empty. But why wouldn't we do this defensive style? As if, for some unforeseen reason, the value wouldn't be json (maybe a faulty custom editor or whatever) we would have an uncaught exception